tests/helpers.bash

```diff
@@ -60,24 +64,18 @@ function teardown() {
 }
 
 function _prefetch() {
-    # Tell podman to use the same mirror registries as buildah, to
-    # avoid docker.io
-    export REGISTRIES_CONFIG_PATH=${TESTSDIR}/registries.conf
-
```

Comment:
I thought we somehow referenced this in Podman, but I'm not finding it....

Reply:
Podman uses it - but we no longer use podman for pulls (See #3099). I should've removed these lines as part of that PR, but didn't notice it at the time. This is just belated cleanup.

And, in any event, the new `podman()` helper function now explicitly includes `$REGISTRY_OPTS` as a command-line argument, which is better than an envariable.